tests/vds/merge_maxnodes.rb

```diff
@@ -85,6 +85,11 @@ def parseStat(data)
   end
 
   def test_merge_max_nodes
+    ['', '2', '3', '4', '5', '6'].each do |n|
+      vespa.adminserver.execute("vespa-logctl distributor#{n}:distributor.bucketdb.updater debug=on")
+    end
+    vespa.adminserver.execute('vespa-logctl container-clustercontroller:com.yahoo.vespa.clustercontroller.core.SystemStateBroadcaster debug=on,spam=on')
+
```

Comment:
I think there is a simpler way. See `vespa.adminserver.logctl("searchnode:config", "debug=on,spam=on")`

Reply:
Done